tests/docker/docker-compose.yml

```
@@ -1,8 +1,8 @@
 version: '3.8'
 
 services:
-  yii-queue-php:
-    container_name: yii-queue-php
+  queue-php:
+    container_name: queue-php
```

```suggestion
    container_name: yii-queue-php
```

It's used in global namespace. Better to keep `yii-` prefix.